torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -31,30 +66,46 @@ std::shared_ptr<FutureMessage> py_rpc_builtin(
     const std::string& opName,
     const py::args& args,
     const py::kwargs& kwargs) {
-  if (opName.rfind("aten", 0) == 0) {
-    // builtin operators.
-    Symbol symbol = Symbol::fromQualString(opName);
-    for (const auto& op : torch::jit::getAllOperatorsFor(symbol)) {
-      try {
-        // FIXME: This is temporary solution. We should at least refactor
-        // ``createStackForSchema`` to avoid throwing an error.
-        Stack stack = torch::jit::createStackForSchema(
-            op->schema(), args, kwargs, c10::nullopt);
+  Stack stack;
+  auto op = match_builtin_op(opName, args, kwargs, stack);
+  return agent.send(dst, ScriptCall(op, std::move(stack)).toMessage());
+}
 
-        return agent.send(dst, ScriptCall(op, std::move(stack)).toMessage());
-      } catch (std::runtime_error) {}
-    }
-  }
+std::shared_ptr<RRef> py_remote_builtin(
+    RpcAgent& agent,
+    const WorkerId& dst,
+    const std::string& opName,
+    const py::args& args,
+    const py::kwargs& kwargs) {
+  Stack stack;
+  auto op = match_builtin_op(opName, args, kwargs, stack);
 
-  AT_ERROR(
-      "Failed to match operator name ",
-      opName,
-      " and arguments "
-      "(args: ",
-      args,
-      ", kwargs: ",
-      kwargs,
-      ") to a builtin operator");
+  auto& ctx = RRefContext::getInstance();
+  if (ctx->getWorkerId() == dst.id_) {
+    auto ownerRRef = ctx->createOwnerRRef<IValue>(dst.id_);
+    agent.send(
+        dst,
+        ScriptRemoteCall(
+            op,
+            std::move(stack),
+            ownerRRef->id().toIValue(),
+            ownerRRef->id().toIValue()
```

Comment:
This means owner rref == fork rref. The message handler defined above asserts they are not equal.

Reply:
this was intended for sending to self, which is not supported yet. Let me remove this for now.